http-server/src/main/java/io/airlift/http/server/HttpLogLayout.java

```diff
@@ -16,10 +17,22 @@
     public String doLayout(HttpRequestEvent event)
     {
         StringBuilder builder = new StringBuilder();
+
+        // format content interarrival time stats
+        String contentInterarrivalStats = null;
+        DoubleSummaryStats stats = event.getResponseContentInterarrivalStats();
+        if (stats != null) {
+            contentInterarrivalStats = format("%.2f, %.2f, %.2f, %d", stats.getMin(), stats.getAverage(), stats.getMax(), stats.getCount());
+        }
+
         builder.append(ISO_FORMATTER.format(event.getTimeStamp()))
                 .append('\t')
                 .append(event.getClientAddress())
                 .append('\t')
+                .append(event.getStreamId())
+                .append('\t')
+                .append(event.getProtocolVersion())
+                .append('\t')
```

Comment:
I don't think we want to reorder the columns, as it will break anyone parsing the file (like the Presto local file connector)

Reply:
Appended the new columns.